File: pkg/server/pokete/msg/position/remove.go
package position

import "github.com/lxgr-linux/pokete/bs_rpc/msg"

const RemoveType msg.Type = "pokete.position.remove"

type Remove struct {
    msg.BaseMsg
    UserName string `json:"user_name"`
}

func (r Remove) GetType() msg.Type {
    return RemoveType
}

func NewRemove(userName string) Remove {
    return Remove{msg.BaseMsg{}, userName}
}

File: src/pokete/classes/multiplayer/msg/position/remove.py
from typing import TypedDict

import pokete.bs_rpc as bs_rpc

REMOVE_TYPE = "pokete.position.remove"


class RemoveData(TypedDict):
    user_name: str


class Remove(bs_rpc.Body):
    def __init__(self, data: RemoveData):
        super().__init__(REMOVE_TYPE, data)
